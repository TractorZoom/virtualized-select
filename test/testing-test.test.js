import React from 'react'
import TestingTest from '../src/testing-test'
import renderer from 'react-test-renderer'

test('Has Testing Div', () => {
    const component = renderer.create(<TestingTest/>)
    let tree = component.toJSON()

    expect(tree).toMatchSnapshot()
})