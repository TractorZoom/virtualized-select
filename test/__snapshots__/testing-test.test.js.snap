// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Has Testing Div 1`] = `
<div>
  Testing Test
</div>
`;
